packages/expo-image/src/Image.types.ts

```diff
@@ -288,7 +288,6 @@ export interface ImageProps extends ViewProps {
    *
    * Downscaling is never used when the `contentFit` prop is set to `none` or `fill`.
    * @default true
-   * @platform android
```

Comment:
Usually I would say that now it should be Android and iOS since we can't control it on web, but I feel like it's kinda obvious and doesn't matter that much.